src/main/webapp/app/modules/conflicts/shared/components/input-field.tsx

```diff
@@ -228,7 +228,7 @@ export class InputField extends React.Component<IInputFieldProp, IInputFieldStat
     let urlButton = null;
     if (fieldName === 'url' && defaultValue) {
       const url = this.props.activity.organization.url;
-      urlButton = (
+      urlButton = url && (
```

Comment:
I've added this because I've encounter 'unexpected error' (startsWith called on null).